### DISCUSSION TITLE
Make northrend glyphs tradeable for badges of justice.

### DISCUSSION THREAD

**Discussion by joesill** [chromiecraft/chromiecraft#6847] [Suggestions]
Badges are a currency and if the other main types of currency, gold and honor points, can be used to buy northrend glyphs, it would be nice to use badges to buy them as well.

It would give people that have no interest in heroic or badge gear a reason to farm them as well, which in itself would make people that complain about queue times happier.

50 badges per glyph. A bargain.

**Comment by balleny** [comment]
given that badges dont drop in raids yet, i would even suggest something like 30 badges to be fair for 1 glyph

I like the idea to give players another way to get them right now.
